stdlib/obj.ml

```diff
@@ -18,8 +18,9 @@ type t
 external repr : 'a -> t = "%identity"
 external obj : t -> 'a = "%identity"
 external magic : 'a -> 'b = "%identity"
-external is_block : t -> bool = "caml_obj_is_block"
 external is_int : t -> bool = "%obj_is_int"
+let is_block a = is_int a |> not
+  [@@ocaml.inline]
```

Comment:
I agree we should standardize, although I don't agree that we should have qualified "ocaml" attributes in the compiler distribution.  It seems like noise to me.  I agree we should always specify "always" explicitly.